src/lib/Components/Inbox/Conversations/Avatar.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from "react"
+
+import { Text, View } from "react-native"
+
+import styled from "styled-components/native"
+import colors from "../../../../data/colors"
+import fonts from "../../../../data/fonts"
+
+const BackgroundCircle = styled.View`
+  background-color: ${colors["gray-regular"]};
+  height: 30
+  width: 30
+  border-radius: 15
+`
+const Name = styled.Text`
+  color: ${props => (props.user ? "white" : colors["gray-semibold"])};
+  font-family: ${fonts["avant-garde-regular"]};
+  font-size: 10
+  align-self: center
+  margin-top: 8
+`
+
+interface Props {
+  senderName: string
+  isUser: boolean
+}
+
+export default class Avatar extends React.Component<Props, any> {
+  render() {
+    const initials: string = this.props.senderName
```

Comment:
yeah you're right. We'll need another one for users, but I will try and implement it there. In which case I'll put this PR on hold :)